scripts/noshake.json

```diff
@@ -203,6 +203,7 @@
   ["Mathlib.Topology.Algebra.UniformRing"],
   "Mathlib.Tactic.Widget.SelectPanelUtils":
   ["ProofWidgets.Component.OfRpcMethod"],
+  "Mathlib.Tactic.Widget.GCongr": ["Mathlib.Algebra.ModEq"],
```

Comment:
I don't know how shake works, is this still needed?

Reply:
No, it is not!
```suggestion